kore/src/Kore/Profiler/Data.hs

```diff
@@ -58,12 +58,15 @@ class Monad profiler => MonadProfiler profiler where
 
 -- Instance for tests.
 instance MonadProfiler Identity where
-    profileDuration _ = id
+    profileDuration = \_ x -> x
```

Comment:
Is this different in any way (performance?), or do you feel it's easier to read?

Reply:
Is this an optimization that, in principle, depends on the compiler version? I.e. would it be reasonable to expect later versions of the compiler to inline `profileDuration _ = id`?

Reply:
I think that is very unlikely. GHC has done inlining this way for a long time (more than ten years) and I don't expect that to change. On a practical level, inlining unsaturated applications isn't a safe optimization: it always introduces a lambda expression and if it isn't reduced away by the optimizer, that lambda becomes an allocation at runtime (possibly a very large one).